src/fsharp/FSharp.Compiler.Service.netcore/project.json

```diff
@@ -0,0 +1,288 @@
+    "../../absil/ilwrite.fs",
+    "../../absil/ilreflect.fs",
+    "../../utils/CompilerLocationUtils.fs",
+    "../PrettyNaming.fs",
+    "../../ilx/ilxsettings.fs",
+    "../../ilx/EraseClosures.fsi",
+    "../../ilx/EraseClosures.fs",
+    "../../ilx/EraseUnions.fsi",
+    "../../ilx/EraseUnions.fs",
+    "../UnicodeLexing.fsi",
+    "../UnicodeLexing.fs",
+    "../layout.fsi",
+    "../layout.fs",
+    "../ast.fs",
+    "pppars.fs",
+    "pars.fs",
+    "../lexhelp.fsi",
+    "../lexhelp.fs",
+    "pplex.fs",
+    "lex.fs",
+    "../LexFilter.fs",
+    "../tainted.fsi",
+    "../tainted.fs",
+    "../ExtensionTyping.fsi",
+    "../ExtensionTyping.fs",
+    "../QuotationPickler.fsi",
+    "../QuotationPickler.fs",
+    "../tast.fs",
+    "../TcGlobals.fs",
+    "../TastOps.fsi",
+    "../TastOps.fs",
+    "../TastPickle.fsi",
+    "../TastPickle.fs",
+    "../import.fsi",
+    "../import.fs",
+    "../infos.fs",
+    "../NicePrint.fs",
+    "../AugmentWithHashCompare.fsi",
+    "../AugmentWithHashCompare.fs",
+    "../NameResolution.fsi",
+    "../NameResolution.fs",
+    "../TypeRelations.fs",
+    "../PatternMatchCompilation.fsi",
+    "../PatternMatchCompilation.fs",
+    "../ConstraintSolver.fsi",
+    "../ConstraintSolver.fs",
+    "../CheckFormatStrings.fsi",
+    "../CheckFormatStrings.fs",
+    "../FindUnsolved.fs",
+    "../QuotationTranslator.fsi",
+    "../QuotationTranslator.fs",
+    "../PostInferenceChecks.fsi",
+    "../PostInferenceChecks.fs",
+    "../TypeChecker.fsi",
+    "../TypeChecker.fs",
+    "../Optimizer.fsi",
+    "../Optimizer.fs",
+    "../DetupleArgs.fsi",
+    "../DetupleArgs.fs",
+    "../InnerLambdasToTopLevelFuncs.fsi",
+    "../InnerLambdasToTopLevelFuncs.fs",
+    "../LowerCallsAndSeqs.fs",
+    "../autobox.fs",
+    "../IlxGen.fsi",
+    "../IlxGen.fs",
+    "../CompileOps.fsi",
+    "../CompileOps.fs",
+    "../CompileOptions.fsi",
+    "../CompileOptions.fs",
+    "../fsc.fsi",
+    "../fsc.fs",
+    "../vs/IncrementalBuild.fsi",
+    "../vs/IncrementalBuild.fs",
+    "../vs/Reactor.fsi",
+    "../vs/Reactor.fs",
+    "../vs/ServiceConstants.fs",
+    "../vs/ServiceDeclarations.fsi",
+    "../vs/ServiceDeclarations.fs",
+    "../vs/Symbols.fsi",
+    "../vs/Symbols.fs",
+    "../vs/Exprs.fsi",
+    "../vs/Exprs.fs",
+    "../vs/ServiceLexing.fsi",
+    "../vs/ServiceLexing.fs",
+    "../vs/ServiceParseTreeWalk.fs",
+    "../vs/ServiceNavigation.fsi",
+    "../vs/ServiceNavigation.fs",
+    "../vs/ServiceParamInfoLocations.fsi",
+    "../vs/ServiceParamInfoLocations.fs",
+    "../vs/ServiceUntypedParse.fsi",
+    "../vs/ServiceUntypedParse.fs",
+    "../vs/service.fsi",
+    "../vs/service.fs",
+    "../vs/SimpleServices.fsi",
+    "../vs/SimpleServices.fs",
+    "../fsi/fsi.fsi",
+    "../fsi/fsi.fs"
+  ],
+  "dependencies": {
+    "Microsoft.FSharp.Core.netcore": "1.0.0-*",
+    "NETStandard.Library": "1.5.0-rc2-23925"
+  },
+  "tools": {
+    "dotnet-fssrgen": "3.0.1",
+    "dotnet-compile-fsc": {
```

Comment:
`dotnet-compile-fsc` mean you use latest version of .net core, you should use version `1.0.0-beta-002071` ( see [docs](https://github.com/enricosada/fsharp-dotnet-cli-samples/wiki/Status) )

We need to pin to one specific version of .net cli (until -rc2 is released), because latest can be broken.